R/configuration-plan.R

```diff
@@ -0,0 +1,117 @@
+#' @title ConfigurationPlan
+#' @docType class
+#' @description  Abstract class for ConfigurationPlan

```

Comment:
Not an abstract class

Also let's use this opportunity to document all parameter of the class right away. It's a pain to do during the dev but too much work to do after the fact

Reply:
see example of documentation of R6Class here
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/quantity.R#L8